.github/workflows/main.yml

```diff
@@ -70,4 +70,5 @@ jobs:
         shell: bash -l -eo pipefail {0}
         run: |
           export TEST_MAMBA_EXE=$(pwd)/build/powerloader
-          pytest powerloader/tests/
+          # pytest powerloader/tests
+          pytest powerloader/test/pytest/run_tests.py
```

Comment:
the workdir is already the repository. 

Just use `pytest ./test`

Reply:
oops, I saw that suggestion after pushing. The error code seems to have changed from four to five now that `pytest ./test` is in use.